frontend/src/api/reports/useReportsPaginated.ts

```diff
@@ -0,0 +1,19 @@
+
+import { ReportQueryParams } from "./reportQueryParameters.ts";
+import api from "../apiClient.ts";
+import { reportQueryKeys } from "./reportQueryKeys.ts";
+import {usePaginatedQuery} from "@/lib/tanstackExtension.ts";
+
+export function useReportsPaginated(params: ReportQueryParams) {
+  const getReportsPaginatedFn = async () => {
```

Comment:
This should be const getReportsPaginatedFn = async(paginatedParams: ReportQueryParams) => {

with const response = await api.get("/reports/", {params: paginatedParams});